### PR TITLE
Privacy Policy

### PR DESCRIPTION
**Pull request by bgregg**
Here it is in all of its glory
screen shot 2016-09-14 at 3 51 03 pm

### CODE REVIEW

**Comment by akaplo** [comment]
⚡  @werebus thoughts?

**Comment by akaplo** [comment]
"...for the purpose of continuing improvements to..." doesn't sound like proper English?

File: www/pages/privacy-policy/privacy-policy.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<ion-view view-title="Privacy Policy">
+  <ion-content>
+    <div class="list card item-text-wrap">
+      <div class="item item-text-center">
+        This application uses your geolocation as well as Google Analytics.
+        The data collected is the property of UMass Transit Services. Google Analytics data is solely used by the developers of this application for the purpose of continuing
```
Comment (akaplo) [inline www/pages/privacy-policy/privacy-policy.html]: "...for the purpose of continuing improvements to..." doesn't sound like proper English?

**Comment by akaplo** [comment]
Otherwise, sounds good to me.

**Comment by bgregg** [comment]
Better?

File: www/pages/about/about.html [hunk 2]
```diff
@@ -24,6 +24,10 @@
         <i class="icon ion-bug"></i>
         Find a bug?  Let us know!
       </a>
+      <a class="item item-icon-left" href="#/app/about/privacy-policy" ngClick="console.log('page rendered')">
```
Comment (werebus) [inline www/pages/about/about.html]: console.log?

File: www/pages/privacy-policy/privacy-policy.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        to improve the application. Google Analytics data is completely anonymous. For more information on Google Analytics, please click
+        <a href="https://support.google.com/analytics/answer/6004245?hl=en">here</a>. Use of this application constitutes an agreement to the above policy.
```
Comment (werebus) [inline www/pages/privacy-policy/privacy-policy.html]: to _this_ policy.

**Review by werebus — COMMENTED** [review]
Also, something that addresses who we'll share the information with.  I was tempted to say "nobody", but it occurs to me that someone at e.g. the PVPC might want to know about the usage.  @sherson?

File: www/pages/about/about.html (hunk 2 above)
Comment (bgregg) [reply www/pages/about/about.html]: Woops...

Comment (sherson) [reply www/pages/about/about.html]: To be on the safe side, how about this?:

> Google Analytics data may be used (1) by the developers of this application for the purpose of continuing to improve the application and (2) by the [PVPC](http://www.pvpc.org/) and [PVTA](http://pvta.com/) for doing service analysis. Google Analytics data is completely anonymous...

**Comment by bgregg** [comment]
Anyone else have any thoughts? If not I will merge it in the next few hours.

**Review by akaplo — APPROVED** [review]

**Pull request merged**